tgnnexplainer/xgraph/method/subgraphx_tg.py

```diff
@@ -310,8 +322,8 @@ def _compute_node_score(self, node, sum_count):
         beta = -3
 
         max_event_idx = max(self.root.coalition)
-        curr_t = self.events['ts'][max_event_idx-1]
-        ts = self.events['ts'][self.events.e_idx.isin(node.coalition)].values
+        curr_t = self.timesteps[max_event_idx-1]
+        ts = self.timesteps[np.isin(self.event_indices, node.coalition)]
```

Comment:
If pandas is slow, we should probably remove them completely and replace with np arrays or tensors 
Probably no time for this now but before we submit to the conference it could make sense